config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,32 @@
 # Be sure to restart your server when you modify this file.
+module ParameterFiltering
+  ATTRIBUTE_WHITELIST = %w[
+    action
+    active
+    code
+    controller
+    created_at
+    deleted_at
+    limit
+    metadata_version
+    process_token
+    recorded_at
+    status
+    updated_at
+  ].freeze
 
-# Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+  WHITELIST_REGEX = /(^|_)ids?|#{Regexp.union(ATTRIBUTE_WHITELIST)}/.freeze
```

Comment:
I think we can include `/(^|_)ids?/` into the constant array above, and this regex can simply be 

`WHITELIST_REGEX = Regexp.union(ATTRIBUTE_WHITELIST).freeze`

Reply:
I want to keep the plain attributes separate from any regex magic, so we can add/remove from that list over time.